UnityProject/Assets/Scripts/HealthV2/Living/PlayerHealthV2.cs

```diff
@@ -119,12 +115,10 @@ protected override void Gib()
 			PlayerMove.PlayerScript.pushPull.VisibleState = false;
 			playerNetworkActions.ServerSpawnPlayerGhost();
 		}
-
-		bool RegisterPlayer.IControlPlayerState.AllowChange(bool rest)
```

Comment:
I fix this in another PR

Reply:
https://github.com/unitystation/unitystation/blob/4afe058c81b47744396ddd1113b7daf3835cc152/UnityProject/Assets/Scripts/HealthV2/Living/PlayerHealthV2.cs#L123